server/repository/src/migrations/v1_01_12/stock_line_barcode_id.rs

```diff
@@ -0,0 +1,11 @@
+use crate::{migrations::sql, StorageConnection};
+
+pub(crate) fn migrate(connection: &StorageConnection) -> anyhow::Result<()> {
+    sql!(
+        connection,
+        r#"
+            ALTER TABLE stock_line ADD barcode_id TEXT REFERENCES barcode(id);
```

Comment:
Hmm, how does this fit with other logic (when selecting barcode when scanning with outbound shipments) ?

Reply:
now that it's available on the `stock_line` there's the option to update how that matching logic works. Will make a new issue for that.